CITATION.cff

```diff
@@ -0,0 +1,29 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: INA Editor
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
+type: software
+authors:
+  - given-names: Sander
+    name-particle: van
+    family-names: Rijn
+    email: s.vanrijn@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0001-6159-041X'
+  - given-names: Ermanno
+    name-particle: Lo
+    family-names: Cascio
+    email: ermanno.locascio@esciencecenter.nl
+    affiliation: Netherlands eScience Center
+    orcid: 'https://orcid.org/0000-0002-7257-8034'
+repository-code: 'https://github.com/ESI-FAR/INA-tool'
+url: 'https://inaeditor.tbm.tudelft.nl'
+keywords:
+  - Institutional Network Analysis
+  - INA
```

Comment:
```suggestion
```
IMO "INA" too much of a generic term, also a bit redundant here.

That said, adding a more generic term could be useful. Don't have a good suggestion, I'm thinking of "social graph" but that probably invokes "social media" in everyone's mind. Maybe we can ask Amineh/Renske for suggestions, and add it later?